ios/Video/RCTVideo.m

```diff
@@ -1735,7 +1735,8 @@ - (BOOL)loadingRequestHandling:(AVAssetResourceLoadingRequest *)loadingRequest {
                 if(self.onGetLicense) {
                   NSString *spcStr = [[NSString alloc] initWithData:spcData encoding:NSASCIIStringEncoding];
```

Comment:
Maybe related:

- https://stackoverflow.com/questions/10178936/nsdata-to-nsstring-returns-null

but `spcStr` is not empty in xcode debugger.